static/src/javascripts/projects/common/modules/experiments/tests/contributions-membership-epic-side-by-side.js

```diff
@@ -0,0 +1,133 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-membership-epic.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/modules/experiments/embed',
+    'common/utils/ajax',
+    'common/modules/commercial/commercial-features'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEpic,
+             robust,
+             arrowRight,
+             config,
+             cookies,
+             embed,
+             ajax,
+             commercialFeatures
+) {
+
+
+    return function () {
+
+        this.id = 'ContributionsMembershipEpicSideBySide';
+        this.start = '2016-11-04';
+        this.expiry = '2016-11-07';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Find out if offering membership and contributions side by side with equal weighting is as effective as just offering membership by itself';
+        this.showForSensitive = true;
+        this.audience = 0;
+        this.audienceOffset = 0.1;
+        this.successMeasure = 'Impressions to number of contributions/supporter signups';
+        this.audienceCriteria = 'All users in US and UK who are not reading articles about Brexit';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'One of the 2 variants proves to be a clear winner';
+        this.canRun = function () {
+            var userHasNeverContributed = !cookies.get('gu.contributions.contrib-timestamp');
+            var worksWellWithPageTemplate = (config.page.contentType === 'Article'); // may render badly on other types
+            var keywords = config.page.keywordIds.split(',');
+            var nonKeywordTagIds = config.page.nonKeywordTagIds.split(',');
+            var isNotAboutBrexit = (keywords.indexOf('politics/eu-referendum') == -1) && (nonKeywordTagIds.indexOf('tone/news') == -1);
```

Comment:
Is this supposed to be the negation of `isAboutBrexit`? If so I think you'll want '||' instead of `&&`.